### DISCUSSION TITLE
How to switch to a specific URL

### DISCUSSION THREAD

**Discussion by Kaze1027** [AutomaApp/automa#1573] [Q&A]
I opened several tabs, Here are URLs:
```
http://1.2.3.4:8010/#/personContent?id=2183556&sdfjlkj&=lkajfi
http://1.2.3.4:8010/#/Group?id=654215221&sfkj=sdjlf
```
How can I use switch tab block to switch to `http://1.2.3.4:8010/#/personContent?id=2183556`, not `http://1.2.3.4:8010/#/Group?id=654215221`

I've read this page `https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns`, and still could not make it, someone help me with it.

**Comment by Delusionnnn** [comment]
> I opened several tabs, Here are URLs:
> 
> ```
> http://1.2.3.4:8010/#/personContent?id=2183556&sdfjlkj&=lkajfi
> http://1.2.3.4:8010/#/Group?id=654215221&sfkj=sdjlf
> ```
> 
> How can I use switch tab block to switch to `http://1.2.3.4:8010/#/personContent?id=2183556`, not `http://1.2.3.4:8010/#/Group?id=654215221`
> 
> I've read this page `https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns`, and still could not make it, someone help me with it.

And this wouldn't work? 

http://1.2.3.4:8010/#/personContent?id=*

**Comment by rsedaghati** [comment]
I've tried different ways. Url regex can't match fragment part (part after #).
I think Automa needs enhancement for matching urls and selecting tabs.
